extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -578,6 +578,11 @@ export async function promptConnectionStringPasswordAndUpdateConnectionString(co
 			// or user chooses to not include password (or if user cancels out of include password prompt), or authentication type is not SQL login
 			connectionString = await vscodeMssqlApi.getConnectionString(connectionDetails, false, false);
 
+			if (connectionInfo.authenticationType !== 'SqlLogin') {
+				// if authentication type is not SQL login, remove password in connection string
```

Comment:
Add a note that this is a temporary fix until STS can be fixed not to include the placeholder if it's not necessary